operator/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+- [9930](https://github.com/grafana/loki/pull/9930) **periklis**: Use PodAntiAffinity for all components
```

Comment:
I forgot to add the changelog line for my PR, can we fix this here? (don't know if this needs a changelog entry though. It fixed an issue that had not appeared yet.

```suggestion
- [9930](https://github.com/grafana/loki/pull/9930) **periklis**: Use PodAntiAffinity for all components
- [9860](https://github.com/grafana/loki/pull/9860) **xperimental**: Fix update of labels and annotations of PodTemplates
```